tests/test_toolargparse.py

```diff
@@ -101,6 +101,62 @@
 outputs: []
 """
 
+script_f = """
```

Comment:
```suggestion
script_int = """
```

(and the same for the others, so it is more understandable?)